src/openzaak/components/documenten/tests/admin/test_eiocanonical_inline_audittrail.py

```diff
@@ -105,7 +105,10 @@ def test_eio_add(self):
 
         get_response = self.app.get(self.change_url)
 
-        form = get_response.form
+        response = get_response.form.submit()
+        self.assertEqual(response.status_code, 200)
+
+        form = response.form
```

Comment:
I have no idea how to get the form error list for the inline, it does does display here

Reply:
You can try `response.context["errors"]` to see all errors in the page. If you want to access inlines you can try `response.context["inline_admin_formsets"]`